### PR TITLE
Corrected typo and extended allowed types for MultiSegmentRecord

### PR DESCRIPTION
**Pull request by agent3gatech**
Corrected misordered arguments in util.lines_to_file. Added 'list' as allowed types for seg_name and seg_len, for when writing MultiSegmentRecords.

### CODE REVIEW

**Comment by bemoody** [comment]
Thanks!

In case it's not clear, here's an example which doesn't work prior to this change:

```
>>> import wfdb
>>> r = wfdb.rdheader('3000003', pn_dir='mimic3wdb/1.0/30/3000003')
>>> r.wrheader()
```

Probably nobody tested this code path, and the logic is hard to follow.  This change should fix the immediate problem.

But I think what we really want is, rather than changing the types in `SEGMENT_SPECS`, we want to add `"seg_name"` and `"seg_len"` to `LIST_FIELDS` (wfdb/io/record.py, line 1675.)

**Comment by briangow** [comment]
Thanks @agent3gatech !

**Review by briangow — APPROVED** [review]
This will solve the problem! I'll open an issue to capture the alternate suggestion by @bemoody .
